src/cbmc/bmc.cpp

```diff
@@ -428,6 +429,12 @@ safety_checkert::resultt bmct::run(
   symex.set_message_handler(get_message_handler());
   symex.options=options;
 
+  {
+    const symbolt *init_symbol;
+    if(!ns.lookup(CPROVER_PREFIX "initialize", init_symbol))
+      symex.mode=init_symbol->mode;
+  }  
+
```

Comment:
Wouldn't we also need this in all other cases that lead to an eventual use of value sets? Enumerating those requires some `git grep`.